### ISSUE TITLE
Move appropriate a11y strings under a11y.translatable

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/joist#1008]
In https://github.com/phetsims/rosetta/issues/132, we expect to make accessibility strings translatable by putting them under a `translatable` key nested under the `a11y` key in the strings JSON files. We are going to eagerly move strings under this key now so that it is easier to support translation of this content in the future.

**Comment by jessegreenberg** [comment]
This is done in the above commit. I reviewed all values in the file under the a11y key and they are all simple enough that translation should be possible when supported by the translation utility. Closing.
